### ISSUE TITLE
add keyboard shift / control / alt key status when sending holding events

### ISSUE THREAD

**Issue by eviltik** [yomotsu/hold-event#5]
Hi @yomotsu

Thank you for your open work.

To handle walk/run in your great camera controls component, using hold event, we need extra keys (shift, alt, ctrl)

```
keyLeft.addEventListener( 'holding', event => {
    if (event.shiftKey) ...
    if (event.altKey) ...
    if (event.ctrlKey) ...
} );
```

Should i PR or perhaps you can do that quickly ?

thank you

**Comment by yomotsu** [comment]
Thank for your request
Actually I cought a cold and can't do that now
Would u mind making a PR please?

**Comment by eviltik** [comment]
let's try !

**Comment by eviltik** [comment]
done, do you want a PR for bumping the version to v0.1.1 too ?

**Comment by yomotsu** [comment]
Sorry for the delay...I almost recovered! (It was covid-19...
I'll take a look at your PR and release another version quickly.

**Comment by yomotsu** [comment]
Your PR has been merged and [v0.2.0](https://www.npmjs.com/package/hold-event) was just released.

**Comment by eviltik** [comment]
`rm covid19 -rf`

thank you !
